.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: ruby
+
+rvm:
+- 2.6.6
+
+addons:
+  postgresql: '13'
+  apt:
+    packages:
+      - postgresql-13
+env:
+  global:
+    - PGPORT=5433
+
+services:
+  - postgresql
+
+# cache:
+#   bundler: true
+#   directories:
+#     - node_modules
+#   yarn: true
+
+install:
+  - bundle install
+  # - nvm install node
```

Comment:
Same as above. Why did you comment these out?

Reply:
ah. commented this stuff out as the build was failing on `node-sass` in the `yarn` command of the install.

@aarondufall do I need this even? since it's a RoR project, would `bundle install` suffice for installing all the dependencies?